crates/ra_hir/src/db.rs

```diff
@@ -19,8 +19,8 @@ use crate::{
     ids::SourceFileItemId,
 };
 
-#[salsa::query_group(HirDatabaseStorage)]
-pub trait HirDatabase: SourceDatabase + AsRef<HirInterner> {
+#[salsa::query_group(PersistentHirDatabaseStorage)]
```

Comment:
@matklad I created a new `Persisten...` here, is it ok?

Reply:
:+1: